src/main/kotlin/org/jitsi/jicofo/bridge/BridgeConfig.kt

```diff
@@ -101,6 +102,12 @@ class BridgeConfig {
             .transformedBy { Duration.ofMillis(it.toMillis() / 2) }
     }
 
+    val brewery: String? by optionalconfig {
```

Comment:
nit: this is the brewery jid (as a string), right?  `breweryJid` or `breweryName`, maybe (since it isn't actually a `Jid`)?

Reply:
Yes, that's the brewery JID as a string. Do you care about the name of the `val`, or the name in the config file?

Reply:
The name of the `val` is what caught my eye, but I suppose both.